src/main/scala/DemoFortis.scala

```diff
@@ -278,6 +314,19 @@ object DemoFortis {
           )
         )
       ),
+      "radio" -> List(
+        ConnectorConfig(
+          "Radio",
```

Comment:
The radio stream is designed to process a single radio feed. For ingesting multiple radio feeds, we should use multiple radio streams. I'm syncing up with @kevinhartman to see how we'd represent that in the current stream-factory paradigm since your ask is really a question for the stream-factories rather than the stream implementations.